libs/blocks/tabs/tabs.js

```diff
@@ -3,37 +3,39 @@
  * https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role
  */
 import { createTag, MILO_EVENTS } from '../../utils/utils.js';
+import { debounce } from '../../utils/action.js';
 
-const isElementInContainerView = (targetEl) => {
-  const rect = targetEl.getBoundingClientRect();
-  const docEl = document.documentElement;
-  return (
-    rect.top >= 0
-    && rect.left >= 0
-    && rect.bottom <= (window.innerHeight || /* c8 ignore next */ docEl.clientHeight)
-    && rect.right <= (window.innerWidth || /* c8 ignore next */ docEl.clientWidth)
-  );
+const isTabInTabListView = (tab, partial = false) => {
+  const tabList = tab.closest('[role="tablist"]');
+  const tabRect = tab.getBoundingClientRect();
+  const tabListRect = tabList.getBoundingClientRect();
+  if (partial) {
+    return Math.round(tabRect.left) < Math.round(tabListRect.right)
+      && Math.round(tabRect.right) > Math.round(tabListRect.left);
```

Comment:
Saving the `Math.round(...)` method calls as variables might help readability, but I imagine naming this is going to be a pain lol.

Reply:
Refactored this section and now using consts.